commands/start-container.ts

```diff
@@ -19,4 +19,11 @@ export function startContainer() {
 
 export function startContainerInteractive() {
     doStartContainer(true);
+}
+
+export function startAzureCLI() {
+    let terminal: vscode.Terminal = vscode.window.createTerminal('Azure CLI');
+    terminal.sendText(`docker run -it --rm azure-xplat-cli:latest`);
+    terminal.show();
+    terminal.sendText(`azure login`);
```

Comment:
Did you want to prompt for login via `az login`? Maybe the login sessions between the 1.0 and 2.0 CLI versions are interoperable, and so it doesn't matter, but I just thought I'd double-check.